app/scripts/modules/netflix/pagerDuty/pagerDutyTag.component.spec.ts

```diff
@@ -0,0 +1,43 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {PagerDutyTagComponent} from './pagerDutyTag.component';
+import {PagerDutyReader} from 'netflix/pagerDuty/pagerDuty.read.service';
+
+describe('PagerDutyTagComponent', () => {
+
+  let component: PagerDutyTagComponent;
+  let fixture: ComponentFixture<PagerDutyTagComponent>;
+  // let element: HTMLElement;
+
+  const pagerDutyReaderStub = {
+    listServices: () => {
+      return [
+        {name: 'name', integration_key: 'integrationKey'}
+      ];
+    }
+  };
+
+  beforeEach(() => {
+    // module('ANGULAR_INJECTOR_MODULE', []).factory('$$angularInjector', ['$injector', function ($injector: ng.auto.IInjectorService) {
+    //   TestBed.configureTestingModule({
+    //     declarations: [PagerDutyTagComponent],
+    //     providers: [
+    //       {provide: '$injector', useValue: $injector},
+    //       {provide: PagerDutyReader, useValue: pagerDutyReaderStub}
+    //     ]
+    //   });
+    //   return TestBed.get(Injector);
+    // }]);
```

Comment:
Works for me.  I have a list of stuff for Pete - they were at ng-nl this week and he mentioned circling back afterwards as they'd been preparing heavily for that.  I've updated our checklist doc with a bunch of stuff (it's at the bottom) if you want some light reading.